chain/ethereum/src/env.rs

```diff
@@ -83,6 +83,9 @@ pub struct EnvVars {
     /// Set by the flag `GRAPH_ETHEREUM_GENESIS_BLOCK_NUMBER`. The default value
     /// is 0.
     pub genesis_block_number: u64,
+    /// The time to wait between polls when using polling block ingestor.
+    /// The value is set in millis and the default is 1000.
+    pub ingestor_polling_interval: Duration,
```

Comment:
Not something for this PR, but we should think about making this a per-chain setting in the config file. Same for reorg threshold